frontend/src/Pages/SignUp.jsx

```diff
@@ -302,6 +319,7 @@ const SignUp = () => {
                   ? true
                   : false
               }
+              onClick={register}    //von Nils
```

Comment:
Das Form hat schon onSubmit, du brauchst kein onClick auf dem Button, weil es der einzige Button in der Form ist. Bitte die onSubmit Funktion nutzen.

Also die Funktion, die von "onSubmit" aufgerufen wird heißt "handleFormSubmit".